lib/base-container.js

```diff
@@ -39,6 +39,12 @@ export default class BaseContainer {
 		return driverHelper.waitTillPresentAndDisplayed( this.driver, this.expectedElementSelector, this.explicitWaitMS );
 	}
 
+	waitRequestsToComplete() {
+		return this.driver.wait( () => {
+			return this.driver.executeScript( 'return jQuery.active === 0' );
+		}, 10000, 'Timed out waiting while AJAX loads' );
```

Comment:
Where did this number come from? How did you decide upon it?
It should be declared at least as a constant and preferably as a config value. You could probably use `this.explicitWaitMs` which I believe has the same intention